src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -244,13 +248,14 @@ public BasicStudyInfos createStudy(UUID sourceStudyUuid, UUID studyUuid, String
             return null;
         }
         LoadFlowParameters sourceLoadFlowParameters = LoadflowService.fromEntity(sourceStudy.getLoadFlowParameters());
+        ShortCircuitParameters copiedShortCircuitParameters = ShortCircuitService.fromEntity(sourceStudy.getShortCircuitParameters());
```

Comment:
copiedShortCircuitParameters could be named sourceShortCircuitParameters to be consistent with sourceLoadFlowParameters

Reply:
It's from the PR I was rebased. I'll see for a change after it's merged